lab_3_ann_retriever/main.py

```diff
@@ -49,7 +52,17 @@ def calculate_distance(query_vector: Vector, document_vector: Vector) -> float |
 
     In case of corrupt input arguments, None is returned.
     """
-
+    if document_vector is None or query_vector is None:
+        return None
+    if not query_vector or not document_vector:
+        return 0.0
+    summary = 0.0
+    for i, value in enumerate(document_vector):
+        summary += (query_vector[i] - value) ** 2
+    distance = summary ** 0.5
+    if not isinstance(distance, float):
```

Comment:
what else, if not None?

Reply:
I can't understand

Reply:
Corrected the comment. Why do you check this type?

Reply:
I removed it